### PR TITLE
feat: provide more information in the `package.json` file

### PR DESCRIPTION
**Pull request by Panquesito7**
## Changes proposed

- Provide a description and more information
  about the project in the `package.json` file.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/691"></a>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,5 +66,24 @@
...
+  },
+  "keywords": [
+    "javascript",
+    "linktree",
+    "socials",
+    "eddiehub",
+    "reactjs",
+    "react"
+  ],
+  "author": "EddieHubCommunity",
```
Comment (krishguptadev) [inline package.json:84]: I think the name here should be **Eddie Jaoude**, but not sure yet.

```suggestion
  "author": {
    "name": "EddieHubCommunity",
    "url": "https://eddiehub.org"
  }
```

File: package.json [hunk 2]
```diff
@@ -66,5 +66,24 @@
   },
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
+  },
+  "description": "LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).",
+  "main": "generate.js",
+  "repository": {
+    "type": "git",
```
Comment (krishguptadev) [inline package.json:73]:
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -66,5 +66,24 @@
...
+    "url": "git+https://github.com/EddieHubCommunity/LinkFree.git"
```
Comment (krishguptadev) [inline package.json:74]:
```suggestion
```

File: package.json [hunk 4]
```diff
@@ -66,5 +66,24 @@
...
+  },
```
Comment (krishguptadev) [inline package.json:75]:
```suggestion
```

File: package.json [hunk 5]
```diff
@@ -66,5 +66,24 @@
   },
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
+  },
+  "description": "LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).",
+  "main": "generate.js",
+  "repository": {
```
Comment (krishguptadev) [inline package.json:72]:
```suggestion
  "repository": "github:EddieHubCommunity/LinkFree",
```

File: package.json [hunk 6]
```diff
@@ -66,5 +66,24 @@
...
+    "type": "git",
+    "url": "git+https://github.com/EddieHubCommunity/LinkFree.git"
+  },
+  "keywords": [
+    "javascript",
```
Comment (krishguptadev) [inline package.json:77]: Umm, not sure if this needs to be added especially when this file is a package.json

```suggestion
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Panquesito7) [reply package.json:84]: URL is already added below/above (forgot where it was).

Comment (krishguptadev) [reply package.json:84]: @Panquesito7 it was linkfree.eddiehub.org not eddiehub.org

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Hmm, can you tell why https://github.com/EddieHubCommunity/LinkFree/pull/691/files#r754744891 and https://github.com/EddieHubCommunity/LinkFree/pull/691/files#r754744679 marked as resolved without fixture?

**Comment by Panquesito7** [comment]
> Hmm, can you tell why https://github.com/EddieHubCommunity/LinkFree/pull/691/files#r754744891 and https://github.com/EddieHubCommunity/LinkFree/pull/691/files#r754744679 marked as resolved without fixture?

I don't see a need for those. They are also confusing, and I don't understand why the changes are needed. 😅

**Comment by krishguptadev** [comment]
> > Hmm, can you tell why https://github.com/EddieHubCommunity/LinkFree/pull/691/files#r754744891 and https://github.com/EddieHubCommunity/LinkFree/pull/691/files#r754744679 marked as resolved without fixture?
> 
> I don't see a need for those. They are also confusing, and I don't understand why the changes are needed. 😅

Hmm, I think `github:username/repo` is recommended better than `{ type: 'git'... }` 🤔 

and I am not sure why do we need `javascript` as a keyword on NPM which literally stands for Node Package Manager 😕

File: package.json [hunk 7]
```diff
@@ -66,5 +66,24 @@
   },
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
+  },
+  "description": "LinkFree Connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).",
```
Comment (schmelto) [inline package.json:70]:
```suggestion
  "description": "LinkFree connect audiences to all of your content with just one link. It is an open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - It was initially created on a YouTube [live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).",
```

**Review by schmelto — APPROVED** [review]

**Pull request merged**